src/init.php

```diff
@@ -86,17 +86,17 @@ function purdue_blocks_cgb_block_assets() { // phpcs:ignore
 	);
 }
 
-function purdue_blocks_block_categories( $categories, $post ) {
-	return array_merge(
-			array(
-					array(
-							'slug' => 'purdue-blocks',
-              'title' => __( 'Purdue Blocks', 'purdue-blocks' )
-					),
-        ),
-        $categories
-	);
-}
+// function purdue_blocks_block_categories( $categories, $post ) {
```

Comment:
instead you should actually delete this old custom category code

Reply:
Thanks.